docs/deploying-to-azure.md

```diff
@@ -466,10 +468,18 @@ Add the connection string to the `application-azure.properties` file:
 quarkus.mongodb.connection-string=mongodb://fights-db:tF8pMgaFm6mgyRSKKpPp3cSEkZAEoFmxpHsqrIJ94vMLWMBTqEvHWw1CyuNPebOfinipMK3qPfKVovQAAGQ5DA==@fights-db.mongo.cosmos.azure.com:10255/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@fights-db@
 ```
 
+As for the microservice invocations, you need to set the URLs of both Heroes and Villains microservices.
+For that, get the values of the `HEROES_URL` and `VILLAINS_URL` variables and add them to the `application-azure.properties` file:
+
+```shell
+quarkus.rest-client.hero-client.url=https://rest-heroes-app.thankfulpond-0365fcc1.eastus2.azurecontainerapps.io
+fight.villain.client-base-url=https://rest-villains-app.thankfulpond-0365fcc1.eastus2.azurecontainerapps.io
+```
```

Comment:
Do you need external fully-qualified URLs? Or is there an internal network where you can just reference the service? Or with Container apps, has all that underlying k8s been abstracted away?

I would think in a series of microservices, not all of them are meant to necessarily be publicly available?